infrastructure/main.tf

```diff
@@ -1,3 +1,11 @@
+provider "vault" {
+  address = "https://vault.reform.hmcts.net:6200"
```

Comment:
Please read this: https://tools.hmcts.net/confluence/display/RPE/Coding+in+the+open#Codingintheopen-Typesofsensitivedataandinformation

Reply:
@bartlabno commented the following:

@Paul Gain as mentioned vault address is not a hostname, and can't be encrypted as vault need to know where to point before it will start decrypting.

https://vault.reform.hmcts.net:6200 is not a hostname but DNS address which doesn't have anything in common. It is mandatory and cannot be encrypted by hashi vault to be decrypted by hashi vault. This is only one vaule which is always in plain text accross all SSCS configs and one of many in other repos.